drivers/scheduler/scheduler.go

```diff
@@ -46,6 +46,16 @@ func (in *Context) GetID() string {
 	return in.App.GetID(in.UID)
 }
 
+// VpsParameters , volume placement strategy parameters
+type VpsParameters struct {
```

Comment:
instead of this, let's just use the placement spec in the schedule options to be consistent with how autopilot uses. With that, all extra schedule options will get converged under the schedule options.